Sources/Sentry/include/SentryHub.h

```diff
@@ -0,0 +1,74 @@
+//
+//  SentryHub.h
+//  Sentry
+//
+//  Created by Klemens Mantzos on 11.11.19.
+//  Copyright © 2019 Sentry. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+#if __has_include(<Sentry/Sentry.h>)
+#import <Sentry/SentryClient.h>
+#import <Sentry/SentryBreadcrumb.h>
+#import <Sentry/SentryDefines.h>
+#import <Sentry/SentryEvent.h>
+#else
+#import "SentryClient.h"
+#import "SentryBreadcrumb.h"
+#import "SentryDefines.h"
+#import "SentryEvent.h"
+#endif
+
+NS_ASSUME_NONNULL_BEGIN
+@interface SentryHub : NSObject
+
++ (instancetype)defaultHub;
+
+- (void)startWithOptions:(NSDictionary<NSString *,id> *)options;
+
+// keeps a stack of client and scope
+
+//- `Hub::new(client, scope)`: Creates a new hub with the given client and scope. The client can be reused between hubs. The scope should be owned by the hub (make a clone if necessary)
```

Comment:
In the final version we should write proper comments, I suppose these are dev helpers.

Reply:
yes. but still, will fix this now.